src/couch/src/couch_file.erl

```diff
@@ -338,12 +339,33 @@ init_status_error(ReturnPid, Ref, Error) ->
     ReturnPid ! {Ref, self(), Error},
     ignore.
 
+last_read(Fd) when is_pid(Fd) ->
```

Comment:
The name of the function is a bit confusing. When I read it first time I thought it would return a timestamp but it returns relative time since last read. Naming is hard. Few options:
 - last_read - i.e. keep it as is
 - seconds_since_read
 - seconds_since_access - if we want to account for both writes and reads
 - seconds_after_read